### DISCUSSION TITLE
Welcome to Neuro Discussions!

### DISCUSSION THREAD

**Discussion by kimjammer** [kimjammer/Neuro#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and potential features.
  * Talk about new models of interest.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by ByronlLove** [comment]
C'est moi ou c'est super dure d'installer les bibliothèques xD
